### ISSUE TITLE
Visual Bugs & Massaging: explore data misc

### ISSUE THREAD

**Issue by ericronne** [18F/doi-extractives-data#1243]
Time permitting, here are some additional visual adjustments which should make for a better read on some of the Explore pages. @gemfarmer and i discussed some of these; just wanted to be sure we have them listed (we can go over this "live" if you think that'd be useful, thx!) ...

 - [x] **Experiment:** change "filter headers" to H2 and the subs further down on those pp to H5 (this second change matches orig design). **Explanation:** The headers in situations like the one here are fighting with each other for prominence, which makes for a slightly ambiguous hierarchy...

 > ![image](https://cloud.githubusercontent.com/assets/9259185/12767382/765f0de8-c9ce-11e5-89c4-5202d9ead320.png)

 > In the original design, heads like "Total revenues by type" are smaller (H5), which would help. But maybe we should do an experiment and go further, exploiting the H2 level as well, like i've done in the example below? This would cause some of these titles to break onto three lines, but that might be ok. (Note: i don't think we have these subheads on any map pp, so we would only have to worry about the map title on those) ...

 > ![image](https://cloud.githubusercontent.com/assets/9259185/12767751/b7c6a906-c9d0-11e5-9307-d36ce3779aae.png)

 - [x] To open things up a bit, it might be nice to scoot the maps down by about 40 pixels....

 > ![image](https://cloud.githubusercontent.com/assets/9259185/12768121/2784ce2e-c9d3-11e5-94f0-4ff3246cf2c4.png)

 - [x] Reduce width of the company filter on the tables and let it nestle partway into the space to the left of the small column headers
 > ![image](https://cloud.githubusercontent.com/assets/9259185/12757255/e572181a-c99c-11e5-8fea-f71c1047ecf1.png)

 - [x] Would also be cool if we included a glossary-like company filter (as we've chatted on previously) ...
 > 
![image](https://cloud.githubusercontent.com/assets/9259185/12769222/99bccc1e-c9db-11e5-9de9-789beac44fdc.png)

 - [x] Reconciliation: center-stack the left ("payments reported by...") header on the table
 > ![image](https://cloud.githubusercontent.com/assets/9259185/12768346/b0036782-c9d4-11e5-8d93-a24117786bb1.png)

 - [ ] Should headers such as these be center stacked over the data column, for consistency?
 > **Current**
![image](https://cloud.githubusercontent.com/assets/9259185/12768691/18792d7c-c9d7-11e5-824f-0e78d6f55af5.png)

 > **Suggested reformatting**
![image](https://cloud.githubusercontent.com/assets/9259185/12768823/35d4e4f0-c9d8-11e5-9cc8-fc071ab234a0.png)

 - [x] We may want to boldface the totals, for consistency and a better scan (Fed rev by co here)
 > 
![image](https://cloud.githubusercontent.com/assets/9259185/12768886/c4515376-c9d8-11e5-935a-aa2ec51ed9e5.png)

 - [x] Remind me again for the umpteenth time and then i swear i'll shut up about it: why show pennies when we're talking millions?
 > 
![image](https://cloud.githubusercontent.com/assets/9259185/12768965/6c7cc0da-c9d9-11e5-9d32-0423d0e25091.png)

 - [ ] Should we consider this collapsing feature (from orig design) for rev by co?
 > 
![image](https://cloud.githubusercontent.com/assets/9259185/12769005/ac411e00-c9d9-11e5-9e0c-b190e292a65a.png)

Apologies again for any redundancies here. thx!

**Comment by ericronne** [comment]
- [ ] The side-by-side bar charts are too close together on GDP and Exports, which makes the number labeling a little hard to follow. Possible to separate them a little more?
![image](https://cloud.githubusercontent.com/assets/9259185/12789263/b9f0ffb4-ca62-11e5-9033-cc53be1af7e8.png)

**Comment by ericronne** [comment]
- [ ] Add space above the "Median discrepancies..." head (and reduce to H5 as noted above, along with the "Discrepancies in payments..." header)

 > 
![image](https://cloud.githubusercontent.com/assets/9259185/12789530/dd878dfc-ca63-11e5-9362-b6a00cd4dc0b.png)

**Comment by gemfarmer** [comment]
@ericronne after reviewing these chart fixes, I think that it might be a good idea to address after merging the current branch, for a few reasons.
* some of these changes aren't trivial (namely the collapsing revenue types of federal-revenue-by-location)
* the current branch has tons of changes already, and because several of these could bubble over into other explorations, it would be a good idea to silo them as much as possible

**Comment by ericronne** [comment]
:+1:

**Comment by gemfarmer** [comment]
Adding this one into the fold from  #1204
- [ ] Blue "water" in "See how it works" teaser should move down to align with the bottom of the wide blue areas, so that no light blue shows below it

**Comment by gemfarmer** [comment]
> @ericronne: Remind me again for the umpteenth time and then i swear i'll shut up about it: why show pennies when we're talking millions?
@meiqimichelle @shawnbot is there a reason for this?

**Comment by meiqimichelle** [comment]
@ericronne @gemfarmer -- nope, no reason! In fact, there's an issue (#1059) to *remove* the pennies with a :+1: from Judy, we just never got to it before launch. Remove away!

**Comment by gemfarmer** [comment]
@ericronne have any of these todos changed? It seems like some of them are no longer relevant.

**Comment by ericronne** [comment]
@gemfarmer i just now went through and checked off the couple that have been dealt with. The rest are still worth a look imo. I intermingled a couple of enhancements in the list (company live-filtering and the revenue by co accordion notion). Those were just random ideas (hopefully good ones!). ty!

**Comment by gemfarmer** [comment]
@ericronne cool. Consolidated checklist for my purposes.

I'm a little confused by what you mean by "company live-filtering and the revenue by co accordion notion)" It already does filter the list when you enter text. Do you mean something else?

- [ ] 40px buffer between map and filter
- [ ] Glossary-like company filter
- [x] Center stack reconciliation left column header
- [ ] Get rid of pennies on reconciliation and revenue by company
- [ ] Upper line rule off on GDP (not on exports)
- [ ] Add space between columns on GDP and Exports
- [ ] Make sure all headers are over the data columns they are describing
- [ ] See how it works blue bottom
- [ ] Bold face totals (rev by company)
- [ ] Reconciliation and Rev by company (add accordion)

**Comment by ericronne** [comment]
We can ignore for now, it was an enhancement idea. This...

![image](https://cloud.githubusercontent.com/assets/9259185/13536229/562a320c-e204-11e5-8339-71ea6ad21a08.png)

**Comment by gemfarmer** [comment]
Ok. I'll update the checklist accordingly. I don't think it would be overly hard to add, but still, can wait for a rainier day.

**Comment by gemfarmer** [comment]
> Should headers such as these be center stacked over the data column, for consistency?

@ericronne I tried this and it hasn't looks strange with a single header centered over the right column. I think I'm going to make a judgement call here and skip this change.

**Comment by ericronne** [comment]
:+1:

**Comment by meiqimichelle** [comment]
If most of these are done, should we move the remaining few into separate issues and close this one?

**Comment by gemfarmer** [comment]
Yes. Doing this now

**Comment by gemfarmer** [comment]
Ok closing. Moved remaining issue to #1317
